src/main/webapp/app/exam/participate/summary/points-summary/exam-points-summary.component.scss

```diff
@@ -9,3 +9,7 @@
     color: white;
     padding: 0.25em 0.5em;
 }
+
+.grading-key {
+    margin-top: 1em;
```

Comment:
Use Bootstrap's `.mt-1` class instead.

Reply:
When I added that class the `span`s with that class were `div`s and I noticed it does not have any effect now. So I removed it altogether.